engine-comparison/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+cd
+sudo gcloud docker -- pull gcr.io/fuzzer-test-suite/gcloud-clang-deps
+sudo docker build -t base-image -f ~/input/FTS/engine-comparison/Dockerfile ~/input
```

Comment:
From docker-build man page:

> This will read the Dockerfile from the directory specified in PATH.  It also sends any other files and directories found in the current directory to the Docker daemon. The contents of this directory would be used by ADD commands found within the Dockerfile.

Sounds like the context is set to CWD from this...  In this case your context would be `~`.